docs/index.rst

```diff
@@ -1,22 +1,58 @@
-.. Crazyswarm documentation master file, created by
-   sphinx-quickstart on Tue Apr 18 20:00:33 2017.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
+.. _introduction:
 
 Welcome to Crazyswarm's documentation!
 ======================================
 
-The Crazyswarm projects allows you to fly a swarm of quadcopters (using Bitcraze Crazyflie 2.0 directly or as control boards)
-in tight, synchronized formations.
-A motion capture system is recommended (VICON, OptiTrack, Qualisy are supported). We successfully flew 49
-Crazyflies using three Crazyradios. An example video for what you can do is shown below:
+The Crazyswarm platform allows you to fly a swarm of
+`Bitcraze Crazyflie 2.x <https://www.bitcraze.io/products/crazyflie-2-1/>`_
+quadcopters in tight, synchronized formations.
+A motion capture system is recommended: VICON, OptiTrack, and Qualisys are supported.
+We successfully flew 49 Crazyflies using three Crazyradios.
+An example video for what you can do is shown below:
 
 .. raw:: html
 
-    <div style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden; max-width: 100%; height: auto;">
+    <div style="position: relative; padding-bottom: 56.25%; margin-bottom: 20pt; height: 0; overflow: hidden; max-width: 100%; height: auto;">
         <iframe src="https://www.youtube.com/embed/D0CrjoYDt9w" frameborder="0" allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe>
     </div>
 
+It is also possible to use the Crazyflie as a control board for other quadcopter hardware.
+
+
+How is Crazyswarm different from Bitcraze's `Crazyflie Python API <https://github.com/bitcraze/crazyflie-lib-python>`_?
+-----------------------------------------------------------------------------------------------------------------------
+
+Both can be used to control several Crazyflies from a Python script.
+Overall, Crazyswarm has more features but is more complicated.
```

Comment:
I am not sure if we have more features. It's a very different feature set. The Bitcraze API also now has `Swarm` support, and of course they support all firmware features, while we only support whatever seems useful for (our) research.

Reply:
line removed.